misc/master.sql

```diff
@@ -184,6 +184,7 @@ CREATE TABLE `libraries` (
   `lastUpdated` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
   `version` int(10) unsigned NOT NULL DEFAULT '0',
   `shardID` smallint(5) unsigned NOT NULL,
+  `hasData` TINYINT( 1 ) NOT NULL DEFAULT '0',
```

Comment:
We need to add this, but it shouldn't be in this PR.

Reply:
Can I push just this one line to master?

Reply:
Sure.